src/pages/download/_downloadPythonPackage.js

```diff
@@ -1,6 +1,7 @@
 import JSZip from "jszip";
 import localforage from "localforage";
 import { saveAs } from "file-saver";
+import Toastify from "toastify-js";
```

Comment:
Can we move all toasting logic in another file (thank god we can import now), then expose some API like `showToast(title, subtitle, position, ...)`?

I think it can even be put in `src/common` or `src/util`